testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorTest.java

```diff
@@ -754,6 +755,22 @@ public void testShowCreateInformationSchemaTable()
                         ")");
     }
 
+    @Test
+    public void testRenameSchema()
+    {
+        if (!hasBehavior(SUPPORTS_RENAME_SCHEMA) || !hasBehavior(SUPPORTS_CREATE_SCHEMA)) {
+            throw new SkipException("create/rename schema is not supported by the connector");
```

Comment:
It'd be good to verify our assumptions here by trying to rename an existing schema and verifying that it fails with an expected exception.

That way the test will find out if some connectors which support rename schema have declared SUPPORTS_RENAME_SCHEMA incorrectly (which can happen if we implement support in some connectors in future and forget to update the declaration).

Reply:
Added separate tests: for renaming existing and created schema.